src/build.ts

```diff
@@ -63,32 +67,40 @@ async function build(): Promise<void> {
 
     const ioType = process.env.AZLE_VERBOSE === 'true' ? 'inherit' : 'pipe';
 
-    if (command === 'install-dfx-extension') {
-        handleInstallDfxExtensionCommand(ioType);
+    if (command === 'extension') {
+        const subCommand = process.argv[3];
 
-        return;
+        if (subCommand === 'install') {
+            handleInstallExtensionCommand(ioType);
+
+            return;
+        }
     }
 
-    if (command === 'install-global-dependencies') {
-        handleInstallGlobalDependenciesCommand();
+    if (command === 'dev') {
```

Comment:
Create handles for the commands, and subcommands should be handled inside of those command handlers...this comment but supersede some aspects of other comments

Reply:
done